qiskit_ibm_provider/ibm_backend.py

```diff
@@ -529,13 +534,28 @@ def _runtime_run(
     ) -> IBMCircuitJob:
         """Runs the runtime program and returns the corresponding job object"""
         hgp_name = self._instance or self.provider._get_hgp().name
+
+        session = self._session
+        if session:
+            if not session.active:
+                raise RuntimeError(f"The session {session.session_id} is closed.")
```

Comment:
Is this the expected behavior? What if something like the following is run, shouldn't the 2nd job run normally with no session? 

```python
backend.open_session()
job = backend.run(circ)
backend.close_session()

job2 = backend.run(circ) <-- should this job run normally, without session
```

Also I think just running `job = backend.run(circ)` should preserve the previous behavior and not use sessions at all

Reply:
I accept your approach. I modified the code and the tests accordingly.